int32 personId
---
bool status
